tests/e2e-leg-10/sandbox-basic/60-unsandbox-sec3.yaml

```diff
@@ -10,7 +10,10 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-
+apiVersion: kuttl.dev/v1beta1
```

Comment:
The changes in this file should be removed since you already extended the timeout in 60-assert.yaml.